stable/openvpn/README.md

```diff
@@ -103,6 +103,7 @@ Parameter | Description | Default
 `openvpn.OVPN_K8S_POD_SUBNET`        | Kubernetes pod network subnet (optional)                             | `255.0.0.0`
 `openvpn.OVPN_K8S_SVC_NETWORK`       | Kubernetes service network (optional)                                | `nil`
 `openvpn.OVPN_K8S_SVC_SUBNET`        | Kubernetes service network subnet (optional)                         | `nil`
+`openvpn.SKIP_DEFAULT_ROUTE`         | Do not push a route which openvpn sets by default (optional)         | `nil`
```

Comment:
can we make this `DEFAULT_ROUTE_ENABLED` and set the default value to `true`?

Reply:
fixed